packages/client/src/components/Actions/components/ActionRequired.js

```diff
@@ -0,0 +1,62 @@
+import { ACTION_TYPES } from "constants/enums";
+import NFTView from "./NFTView";
+import TokenView from "./TokenView";
+import OwnerView from "./OwnerView";
+import ThresholdView from "./ThresholdView";
+import Confirmations from "./Confirmations";
+
+const ActionRequired = ({
+  safeData,
+  actionView,
+  confirmations,
+  onReject,
+  onApprove,
+}) => {
+  const actionType = actionView.type;
```

Comment:
Prefer use destructuring
```suggestion
  const {type: actionType } = actionView
```